.drone.yml

```diff
@@ -3,13 +3,42 @@ workspace:
   path: src/github.com/drone-plugins/drone-docker
 
 pipeline:
-  build:
-    image: golang:1.9
+  dep:
+    image: golang:1.11
     commands:
       - go get -u github.com/golang/dep/cmd/dep
       - dep ensure
-      - go vet ./...
+
+  fmt:
+    image: golang:1.11
+    group: validate
+    commands:
+      - GOFILES_NOVENDOR=$(find . -type f -name '*.go' -not -path "./vendor/*")
```

Comment:
I'm pretty sure this is not required with 1.11.

Reply:
Still required:

```
$ gofmt -l -s -e .
vendor/github.com/aws/aws-sdk-go/aws/corehandlers/param_validator_test.go
...
```